lib/components/viewers/stop-time-cell.js

```diff
@@ -50,18 +47,17 @@ const StopTimeCell = ({
 
   // Determine whether to show departure as countdown (e.g. "5 min") or as HH:mm
   // time, using realtime updates if available.
-  const secondsUntilDeparture = getSecondsUntilDeparture(stopTime, false)
+  const secondsUntilDeparture = Math.round(getSecondsUntilDeparture(stopTime, false))
   // Determine if vehicle arrives after midnight in order to advance the day of
   // the week when showing arrival time/day.
   const departsInFuture = secondsUntilDeparture > 0
   // Show the exact time if the departure happens within an hour.
   const showCountdown = secondsUntilDeparture < ONE_HOUR_IN_SECONDS && departsInFuture
 
   // Use "soon text" (e.g., Due) if vehicle is approaching.
-  const countdownString = secondsUntilDeparture < 60
-    ? soonText
-    : formatDuration(secondsUntilDeparture)
-  const formattedTime = formatSecondsAfterMidnight(departureTime, timeFormat)
+  // Convert seconds after midnight to unix (millisceonds) for FormattedTime
+  var current = new Date()
+  var startOfDay = new Date(current.getFullYear(), current.getMonth(), current.getDate())
```

Comment:
These should be `const`. Also because they are repeat of another code block, that could be extracted to a function.

Reply:
Changed to `const` in new methodology. The use in `trip-viewer` is slightly different but lmk and I can refactor into a function.